Resources/Private/Partials/ContentElements/Socialmedia.html

```diff
@@ -4,37 +4,101 @@
 	<f:section name="Default">
 		<!-- theme_t3kit: Partials/Footer/Socialmedia.html [begin] -->
 		<div class="social-icons">
-			<f:if condition="{theme:constant(constant:'themes.configuration.socialmedia.useFacebook')}">
-				<a class="social-icons__item" href="{theme:constant(constant:'themes.configuration.socialmedia.facebookLink')}"
-				   target="_blank" title="Facebook"><span class="icons icon-t3-facebook"></span></a>
+			<f:if condition="{settings.facebookLink}">
```

Comment:
@thea7y this block of code should move into `<f:if condition="{theme:constant(constant:'themes.configuration.socialmedia.useFacebook')}">` for all social links.